az-core/src/main/java/azkaban/Constants.java

```diff
@@ -233,8 +233,9 @@
 
     public static final String SESSION_TIME_TO_LIVE = "session.time.to.live";
 
-    // allowed max size of shared project dir in MB
-    public static final String PROJECT_DIR_MAX_SIZE_IN_MB = "azkaban.project_cache_max_size_in_mb";
```

Comment:
given the feature has been documented(https://github.com/azkaban/azkaban/issues/2095), I guess no one outside is using it. But it exposes the problem with our OSS release process - which version should we claim as production ready. Maybe we need to internally exercise the release first and then claim it to be stable version.

Reply:
Agreed that it would be good to have a more official release process, so that it is clear which versions are still in development and subject to change, and which are production ready.